WalletWasabi/Blockchain/Keys/HdPubKey.cs

```diff
@@ -11,18 +11,15 @@
 namespace WalletWasabi.Blockchain.Keys;
 
 [JsonObject(MemberSerialization.OptIn)]
-public class HdPubKey : NotifyPropertyChangedBase, IEquatable<HdPubKey>
```

Comment:
Why was `NotifyPropertyChangedBase` removed?

Reply:
Only because it seems to me it is not used anywhere. If it's not true, we can easily revert https://github.com/zkSNACKs/WalletWasabi/pull/7932/commits/d24c7d264fe8c9489d3558ac8569c77eb57d029a.

Reply:
The UI is subscribed to the notifications of the changes in these. For example: address reuse happens, anonscore goes from 9000 to 1 and the UI updates the privacy progress percentage tile.

Reply:
OK, good to know, I missed that.

Reverting in https://github.com/zkSNACKs/WalletWasabi/pull/7932/commits/7b685a795b8b0e76c51e83ba6e804dec8981d565.